crates/wit-bindgen-wasmcloud-provider/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "wit-bindgen-wasmcloud-provider"
+version = "0.1.0"
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+publish = false
+description = """
+Bindgen macro for binary-based wasmCloud capability providers written in Rust
+"""
+
+[lib]
+proc-macro = true
+
+[dependencies]
+anyhow = { workspace = true }
+heck = { workspace = true }
+proc-macro2 = { workspace = true }
+quote = { workspace = true }
+serde = { workspace = true }
+syn = { version = "1.0.92", features = [ "parsing", "full", "visit-mut", "extra-traits" ] }
+wasm-metadata = "0.10.1"
+wasmtime-component-macro = { version = "11.0.1", features = [ "async" ] }
+wasmtime-wit-bindgen = "11.0.1"
+wit-parser = "0.8.0"
```

Comment:
These should have been workspace dependencies. I'm fine with fixing that later, but could you add a `TODO` here to fix that? Preferably, with a reference to an issue (which you could maybe file?)

Reply:
Ah these deps aren't used by any other crates yet -- is the rule here that all deps need to be hoisted?

Reply:
Currently, that's how it's done for maintenance - all versions are defined at a single location

Reply:
note also that the `syn` dep *cannot* use the top level one -- others can, though.